src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json

```diff
@@ -63,6 +63,7 @@
             "deviceDeleted": "Das Zahlungsmittel wurde erfolgreich entfernt.",
             "title": "Ihre gespeicherten Zahlungsmittel"
         },
-        "finishPaymentFailed": "<a href=\"%editOrderUrl%\">Zahlung ändern</a>"
+        "finishPaymentFailed": "<a href=\"%editOrderUrl%\">Zahlung ändern</a>",
+        "birthdayInvalid": "Geburtsdatum ungültig"
```

Comment:
What about "You need to be at least 18 years of age".

Reply:
Do you only want to change the general error message or do we need to implement multiple cases for displaying different error messages?

Reply:
Please do two distinct checks and error messages.
1. Invalid date (from format, invalid date e.g. 30.2.2001)
2. Buyer is too young (younger than 18 years)

Reply:
@sixer1182 As discussed: We've just changed the text to the second message.